src/client/mod.rs

```diff
@@ -155,6 +155,12 @@ pub struct ClientConfig {
     pub conn_init_thread_num: usize,
     pub query_concurrency_limit: Option<usize>,
 
+    pub conn_reader_thread_num: usize,
+    pub conn_writer_thread_num: usize,
+    pub default_thread_num: usize,
+
+    pub conn_sender_mpsc_channel: usize,
```

Comment:
Maybe we need a more readable name for this option:
```suggestion
    pub max_reqs_in_flight_per_conn: usize,
```

Reply:
Done, using max_inflight_reqs_per_conn instead.